apps/course/src/app/features/legal/legal-detail/legal-markdown-renderer/legal-markdown-renderer.component.ts

```diff
@@ -0,0 +1,55 @@
+import {
+  Component,
+  Input,
+  OnInit,
+  SecurityContext,
+  ViewEncapsulation,
+} from '@angular/core';
+import {
+  CLIPBOARD_OPTIONS,
+  ClipboardButtonComponent,
+  MarkdownComponent,
+  MarkdownService,
+  provideMarkdown,
+} from 'ngx-markdown';
+import { HttpClient } from '@angular/common/http';
+import { FooterComponent } from '../../../../shared/navigation/footer/footer.component';
+import { Legal } from '../../../../types/legal.type';
+import { LoadingBarsComponent } from '../../../../shared/feedback/loading-bars/loading-bars.component';
+
+@Component({
+  selector: 'app-legal-markdown-renderer',
+  standalone: true,
+  imports: [FooterComponent, LoadingBarsComponent, MarkdownComponent],
+  providers: [
+    provideMarkdown({
+      loader: HttpClient,
+      sanitize: SecurityContext.NONE,
+      clipboardOptions: {
+        provide: CLIPBOARD_OPTIONS,
+        useValue: {
+          buttonComponent: ClipboardButtonComponent,
+        },
+      },
+    }),
+  ],
+  templateUrl: './legal-markdown-renderer.component.html',
+  styleUrls: [
+    './legal-markdown-renderer.component.scss',
+    '../../../../styles/markdown.scss',
```

Comment:
`markdown.scss` is added in `angular.json` and doesn't have to be imported anymore.